packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -324,7 +324,7 @@ export class AttributesController extends ConsumptionBaseController {
 
     public async deleteAttributesExchangedWithPeer(peer: CoreAddress): Promise<void> {
         const attributes = await this.getLocalAttributes({ "shareInfo.peer": peer.toString() });
-        await Promise.all(attributes.map((attribute) => this.deleteAttributeUnsafe(attribute.id)));
+        attributes.forEach(async (attribute) => await this.deleteAttributeUnsafe(attribute.id));
```

Comment:
@Magnus-Kuhn please don't use .forEach for async operations

Reply:
these operations will just run in parallel and run into an unhandled promise rejection on an error